fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/FedoraSession.java

```diff
@@ -36,9 +38,26 @@
     void expire();
 
     /**
+     * Save the session
+     *
+     * Note: use of the save method in the context of a batch operation may not persist
+     * changes to the repository until the commit method has been issued.
+     */
+    void save();
+
+    /**
      * Commit any batch operations
      */
-    void commit();
+    default void commit() {
```

Comment:
Perhaps we could remove the `commit` method and replace it with something like `finalize` or `persist`? Either way, using a default method is wrong here.